deno.json

```diff
@@ -1,7 +1,10 @@
 {
   "tasks": {
-    "test": "deno test --unstable --allow-read=worker.ts --trace-ops --fail-fast --coverage=cov_profile && deno coverage cov_profile",
+    "types": "deno types --unstable-worker-options > lib.deno.d.ts",
     "version": "deno run -A https://deno.land/x/version@v1.1.1/index.ts",
-    "push": "git push --atomic origin master v$(deno task version get)"
+    "push": "git push --atomic origin master v$(deno task version get)",
+    "test": "deno task test-net && deno task test-no-net && deno test --unstable-worker-options --allow-env --allow-read=worker.ts --fail-fast --no-check --coverage=cov_profile && deno coverage cov_profile",
+    "test-net": "NET=1 deno test --unstable-worker-options --allow-env --filter=\"network permission allowed\" --allow-read=worker.ts --allow-net=jsonplaceholder.typicode.com:443 --fail-fast --no-check",
+    "test-no-net": "NO_NET=1 deno test --unstable-worker-options --allow-env --filter=\"network permission denied\" --allow-read=worker.ts --allow-net=jsonplaceholder.typicode.com:443 --fail-fast --no-check"
```

Comment:
* Same as above.
* Since this test shouldn't require network access, let's remove the `--allow-net` permission.